common/sw-update/versions-db-manager.h

```diff
@@ -44,7 +44,7 @@ namespace rs2
             {
                 constexpr int MINIMAL_MATCH_SECTIONS = 4;
                 constexpr int MATCH_SECTIONS_INC_BUILD_NUM = 5;
-                std::regex rgx("^(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})(\\.\\d{1,5})?$");
+                std::regex rgx("^(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,5})?$");
```

Comment:
The build number is optional -- and therefore so is the period before it... so I don't understand.

Reply:
fixed